### ISSUE TITLE
Upstream Nomenclature

### ISSUE THREAD

**Issue by blaggacao** [divnix/nix-book#20]
There may be an upstream nomenclature emerging. When it does, shall we adopt?

Let me find the link, again...

**Comment by blaggacao** [comment]
Here's the link: https://github.com/NixOS/nix.dev/pull/263

**Comment by nrdxp** [comment]
Indeed, we just had a meeting about this, and here is a pretty good outline of what was discussed:
https://github.com/NixOS/nix.dev/issues/275#issuecomment-1227572104

**Comment by davidak** [comment]
The latest link is: https://nix.dev/reference/glossary
